### PR TITLE
Add READMEs for markdown and iron-icons packages

### PR DESCRIPTION
**Pull request by chrnorm**
I added basic READMEs with install instructions to `@nteract/markdown` and `@nteract/iron-icons`, to tackle some of #1577.

For the `markdown` package, I renamed @captainsafia's great `examples.md` file to be the README and added install instructions to it - so more people will see it on github and npm.

- [x ] I have read the [Contributor Guide](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by captainsafia** [comment]
Thanks for opening this PR!

Since you changed the name of the file for our markdown docs, you’ll also need to update the file path in the styleguidist config located in the root of the repo.

Other than that, it looks good.

**Comment by chrnorm** [comment]
Ah - I didn't realise that `examples.md` was used for documentation! I restored the file and made the README link to `examples.md` in the "Usage" section. That way, there won't be any `yarn install` instructions etc in the styleguidist docs.

@captainsafia, are the styleguidist docs deployed to components.nteract.io? Or a different site? I was just looking on components.nteract.io and couldn't find the `MarkdownRender` documentation found in `packages/markdown/examples.md`

**Comment by captainsafia** [comment]
@chrnorm Styleguidist is deployed to that URL. Unfortunately, we haven't figured out how to set automatic deployments to [now](https://zeit.co/now) on a monorepo. As a result, the docs have to be manually deployed.

It's been a while since we've pushed an update, mostly because the code base has been in flux due to a TypeScript migration, so the updated changes aren't live. I'll see if it is in a good place to deploy now.

TL;DR: A deploy hasn't happened in a while due to the TypeScript migration so the components site is a little out of date.

**Comment by captainsafia** [comment]
I'm gonna go ahead and merge this. I've opened up a PR that cleans up some new errors that emerged in styleguidist during the migration. Once it is merged, I will redeploy the docs page and you should see the docs for @nteract/markdown live there.

Thanks again for submitting this PR!

**Comment by chrnorm** [comment]
Thanks for clarifying @captainsafia!

**Comment by captainsafia** [comment]
Alright! The redeploy is done. [Here[(https://components.nteract.io/#nteractmarkdown) is a link to the markdown docs.
